runtime/src/lib.rs

```diff
@@ -206,28 +183,13 @@ pub trait Spawner: Clone + Send + Sync + 'static {
     ///
     /// # Warning
     ///
-    /// Blocking tasks cannot be aborted.
-    fn spawn_blocking<F, T>(self, dedicated: bool, f: F) -> Handle<T>
+    /// Blocking tasks cannot be aborted or supervised. Calling this method with [`Spawner::supervised`]
+    /// is a no-op.
```

Comment:
"no-op" in the sense that it will spawn a blocking task unsupervised, or "no-op" in the sense that no task is spawned at all?